src/Lykke.Job.TransactionHandler/Queues/EthereumEventsQueue.cs

```diff
@@ -11,101 +12,109 @@
 
 namespace Lykke.Job.TransactionHandler.Queues
 {
-    public sealed class EthereumEventsQueue : IQueueSubscriber
+    public sealed class EthereumEventsQueue : IStartable, IDisposable
     {
         private const string QueueName = "lykke.transactionhandler.ethereum.events";
         private const string HotWalletQueueName = "lykke.transactionhandler.ethereum.hotwallet.events";
 
         private readonly ILog _log;
+        private readonly string _mongoConnectionString;
```

Comment:
I would prefer to use EthRabbitMqSettings and MongoDeduplicator instead of lot of strings.